testing/smoke/basic-upgrade/test.sh

```diff
@@ -18,7 +20,14 @@ MINOR_VERSION=$(echo ${VERSION} | cut -d '.' -f2 )
 
 if [[ ${MAJOR_VERSION} -eq 7 ]]; then
     ASSERT_EVENTS_FUNC=legacy_assertions
-    LATEST_VERSION=$(curl -s --fail https://artifacts-api.elastic.co/v1/versions/${MAJOR_VERSION}.${MINOR_VERSION} | jq -r '.version.builds[0].version')
+
+    # Check if the version is available.
+    if ! curl -s --fail ${ARTIFACTS_API}/versions/${MAJOR_VERSION}.${MINOR_VERSION} ; then
+        echo "-> Skipping there are no new artifacts to be downloaded in artifacts-api.elastic.co ..."
+        exit 0
+    fi
+
+    LATEST_VERSION=$(curl -s --fail ${ARTIFACTS_API}/versions/${MAJOR_VERSION}.${MINOR_VERSION} | jq -r '.version.builds[0].version')
```

Comment:
It should be possible to check if the `LATEST_VERSION` is empty or not to trigger a skip.
This will avoid a double call to the artifacts-api.